server/src/main/java/org/elasticsearch/search/aggregations/metrics/MultiValueAggregation.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.search.aggregations.metrics;
+
+import org.elasticsearch.search.aggregations.Aggregation;
+
+public interface MultiValueAggregation extends Aggregation {
```

Comment:
This "InternalX" class and "X" interface has existed for a long time for the transport client. But with it deprecated I think we can do away with the "X" interface.

Reply:
Thinking out loud, as much as I'd like to remove these now we probably should keep making them until we're ready to change all of them....